src/slot2/GBACartSuperCardCF.h

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright (c) 2019 Adrian "asie" Siekierka
+
+    This file is part of melonDS.
+
+    melonDS is free software: you can redistribute it and/or modify it under
+    the terms of the GNU General Public License as published by the Free
+    Software Foundation, either version 3 of the License, or (at your option)
+    any later version.
+
+    melonDS is distributed in the hope that it will be useful, but WITHOUT ANY
+    WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+    FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with melonDS. If not, see http://www.gnu.org/licenses/.
+*/
+
+#include "GBACart.h"
+#include "IDEDrive.h"
+
+class GBACartSuperCardCF: public GBACart {
+public:
+    GBACartSuperCardCF(const char *drivePath);
+    ~GBACartSuperCardCF();
+    bool IsValid() { return this->Valid; }
+    virtual u16 RomReadWord(u32 addr);
```

Comment:
16-bit word is a bit ambigious with a 32-bit processor. Maybe rename it to just RomRead/`RomRead16`?